client/src/lib/appState/state/state.ts

```diff
@@ -44,7 +44,10 @@ const useAppState = create<State & Actions>()(
     {
       name: 'appState',
       getStorage: () => AsyncStorage,
-      partialize: ({settings}) => ({settings}),
+      partialize: ({settings, showHiddenContent}) => ({
+        settings,
+        showHiddenContent,
```

Comment:
How about moving this into `settings` instead? :)

Reply:
👍